libstb/secvar/backend/edk2-compat-util.h

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: Apache-2.0
+/* Copyright 2019 IBM Corp. */
+
+#ifndef __SECVAR_EDK2_COMAPT__
+#define __SECVAR_EDK2_COMAPT__
```

Comment:
`__SECVAR_EDK2_COMPAT__`

Probably should be updated to include `UTIL` as well

Reply:
Done